src/test/test-execute.c

```diff
@@ -551,8 +559,14 @@ static void test_exec_capabilityambientset(Manager *m) {
                 return;
         }
 
-        test(m, "exec-capabilityambientset.service", 0, CLD_EXITED);
-        test(m, "exec-capabilityambientset-merge.service", 0, CLD_EXITED);
+        if (have_effective_cap(CAP_NET_ADMIN) <= 0 ||
+            have_effective_cap(CAP_NET_RAW) <= 0) {
+                log_error("Skipping %s, this process does not have enough capabilities", __func__);
+                return;
+        }
+
+        test(m, "exec-ambientcapabilities.service", 0, CLD_EXITED);
```

Comment:
When I first read the code, I didn't notice the unit files had been renamed. I think it would be great if the commit message contained a more detailed explanation of the change :-)

Reply:
Basically, I can't seem to understand why it was necessary to rename the unit files. The names of all the other unit files related to capabilities begin with `exec-capability`.